src/Components/Browser.JS/src/Boot.Server.ts

```diff
@@ -20,25 +21,56 @@ async function boot() {
 
   const initialConnection = await initializeConnection(circuitHandlers);
 
+  var circuitIds: string[] = [];
```

Comment:
`var` -> `const` or `let` everywhere

We should probably have tslint enabled for this.

Reply:
Yeah, this needs cleanup.

Reply:
I enabled eslint